model/pod/db_test.go

```diff
@@ -1 +1,41 @@
 package pod
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/utility"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindByInitializing(t *testing.T) {
+	require.NoError(t, db.Clear(Collection))
```

Comment:
Can you also do this:
```go
defer func() {
    assert.NoError(t, db.Clear(Collection))
}()
```
This ensure that the collection is cleaned up after the test is finished as well.